client/go/outline/vpn_linux.go

```diff
@@ -47,18 +59,37 @@ func establishVPN(configStr string) error {
 
 	tcp := newFWMarkProtectedTCPDialer(conf.VPN.ProtectionMark)
 	udp := newFWMarkProtectedUDPDialer(conf.VPN.ProtectionMark)
-	c, err := NewClientWithBaseDialers(conf.Client, tcp, udp)
+	api.client, err = NewClientWithBaseDialers(conf.Client, tcp, udp)
```

Comment:
We need to make sure api.client is `nil` before starting the session. We allows JavaScript to call `Establish` and another `Establish` without calling `Close`. The second `Establish` needs to make sure it calls the `Close` of the 1st `Establish`.